src/Pages/ProductDetail/Components/RecentlyViewed.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+
+export class RecentlyViewed extends Component {
```

Comment:
```suggestion
export default class RecentlyViewed extends Component {
```
이렇게 작성해주면 하단에서 export해줄 필요가 없겠죠?

Reply:
다른 컴포넌트들도 동일하게 전부 사용할 수 있는건가요??